spec/util/object_storage/miq_swift_storage_spec.rb

```diff
@@ -0,0 +1,68 @@
+require "util/object_storage/miq_swift_storage"
+
+describe MiqSwiftStorage do
+  context "public methods" do
```

Comment:
Can you make this context say something meaningful about the context in which the specs are running?

In this case it seems to be something like `"using a uri without query parameters"`

Reply:
Sure.